modules/javafx.graphics/src/main/java/javafx/scene/robot/Robot.java

```diff
@@ -265,8 +266,8 @@ public Color getPixelColor(Point2D location) {
     /**
      * Returns a {@link WritableImage} containing the specified rectangular area relative
      * to the primary screen. If the given {@code image} is {@literal null}, or if the given
-     * {@code image} is too small, a new {@code WritableImage} will be created and returned.
-     * Otherwise, the given {@code image} is re-used.
+     * {@code image} is not the required size, a new {@code WritableImage} will be created
+     * and returned. Otherwise, the given {@code image} is re-used.
```

Comment:
There are three other variants of getScreenCapture that could use this same wording change.

Reply:
Whoops, of course - just woke up.

Reply:
Once this is done, I'll do one final quick check, and then merge the PR.